components/daisy/Carousel.tsx

```diff
@@ -0,0 +1,68 @@
+export interface Props {
```

Comment:
This component already exists and it's the SliderJS. Remove it, if you agree with.